noun/imprison.c

```diff
@@ -9,7 +9,7 @@
 */
 u3_noun
 u3i_words(c3_w        a_w,
-            const c3_w* b_w)
+          const c3_w* b_w)
```

Comment:
I'm not sure how to review arbitrary unrelated whitespace cleanup. It's certainly a good thing to do, but it seems like it should probably be done in one big pass. Or at least in separate commits.